l10n_es_facturae/data/l10n_es_facturae_data.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate='1'>
+
+        <record id="payment_mode_face_code_01" model="payment.mode.face.code">
```

Comment:
Es igual de flexible, ya que el mantenimiento del módulo corre de nuestra parte, y nadie va a añadir elementos no listados aquí, pero en esto no me voy a enrocar y se puede quedar así. Luego me diréis las incidencias que os surjen porque alguien cree algún valor en este dato (que se puede prevenir poniendo el widget selection en el formulario de fctura), o porque cambien un código (que ya no es prevenible).